mesonbuild/build.py

```diff
@@ -564,6 +567,11 @@ def process_kwargs(self, kwargs, environment):
         if not isinstance(clist, list):
             clist = [clist]
         self.add_compiler_args('c', clist)
+        cstd = kwargs.get('c_std', None)
+        if not isinstance(cstd, str) and cstd != None:
+            raise InvalidArguments('c_std argument must be a string.')
+        if cstd != None:
```

Comment:
__[flake8]__
* [E711] comparison to None should be 'if cond is not None:'

<!-- Comment by [SideCI](https://sideci.com) -->